MIP 9 Community Greenlight Poll - FEI (Fei USD) - May 16, 2022.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - FEI (Fei USD) - May 16, 2022
+summary: Signal your support or opposition to prioritising onboarding FEI (Fei USD).
+discussion_link: https://forum.makerdao.com/t/fei-mip6-collateral-onboarding-application/14895
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Greenlight
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2002-05-16T16:00:00
+end_date: 2002-05-30T16:00:00
+---
+# Poll: Community Greenlight Poll - FEI (Fei USD) - May 16, 2022
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, May 16, at 16:00 UTC.
```

Comment:
remove the second comma, so the end should be: "beginning on Monday, May 16 at 16:00 UTC."